main.tf

```diff
@@ -144,6 +144,10 @@ su - tableau_srv
 
 echo "#Checking PATH"
 echo $PATH
+echo "#sourcing tableau server envs - even though they should already be present (but they are not)"
+source /etc/profile.d/tableau_server.sh
```

Comment:
Should it not be `/home/tableau_srv/env_vars.sh` ?

Reply:
No.
That script is sourced from tableau_srv's .bashrc:
```
echo "#Store env vars in script"
echo "
source /home/tableau_srv/env_vars.sh
" >> /home/tableau_srv/.bashrc
```

This script (tableau_server.sh) should not be required (because after the deploy, when we log in to tableau_srv the path IS set correctly) but is required (as witnessed by outputting $PATH in user_data.log).

Reply:
Ok I don't see `/etc/profile.d/tableau_server.sh` in the config so I take this is a default file created by the Tableau install - if that's the case: LGTM